### DISCUSSION TITLE
de/2-07-kata-6/

### DISCUSSION THREAD

**Discussion by giscus** [cogneon/lernos-ai#64] [General]
# de/2-07-kata-6/

https://ai.lernos.org/de/2-07-kata-6/

**Comment by rednaxelaesor** [comment]
Ich würde den 'call to action' -> "Aufgabe: Starte einen ersten Versuch zur Umsetzung" bestimmter formulieren. Der aktuelle Aufruf klingt für mich etwas "zögerlich" Vorschlag: "Starte mir der Umsetzung"
